pkg/config/config_template.yaml

```diff
@@ -478,16 +478,21 @@ api_key:
 #
 # azure_hostname_style: "os"
 
-## @param flare_stripped_keys - list of strings - optional
-## @env DD_FLARE_STRIPPED_KEYS - space separated list of strings - optional
-## By default, the Agent removes known sensitive keys from Agent and Integrations yaml configs before
-## including them in the flare.
-## Use this parameter to define additional sensitive keys that the Agent should scrub from
-## the yaml files included in the flare.
-#
-# flare_stripped_keys:
-#   - "sensitive_key_1"
-#   - "sensitive_key_2"
+## @param scrubber - custom object - optional
+## Add any parametter you want to scrub.
+#
+# scrubber:
+#
+  ## @param scrubber.additional_keys - list of strings - optional
+  ## @env DD_SCRUBBER_ADDITIONAL_KEYS - space separated list of strings - optional
+  ## By default, the Agent removes known sensitive keys from Agent and Integrations yaml configs before
+  ## including them in the flare.
+  ## Use this parameter to define additional sensitive keys that the Agent should scrub from
+  ## the yaml files included in the flare.
+  #
+  # additional_keys:
```

Comment:
Would `additional_config_entry` be more precise ? :shrug:

Reply:
I took the name that @/vignesh put in the jira card but that something we can ask on the daily ?